CONTRIBUTING.md

```diff
@@ -12,3 +12,23 @@ The base version represents the MAJOR and MINOR parts of [SemVer](https://semver
 
 - `main` contains the latest sources. Each merge there triggers a deploy to `nuget.org`.
 - All versions on `nuget.org` have a matching GitHub release/tag.
+
+## Commits and PR title
+
+- Please use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) to name your commits and PR title
```

Comment:
As we use [amannn/action-semantic-pull-request](https://github.com/amannn/action-semantic-pull-request?tab=readme-ov-file#configuration) for enforcing this policy, maybe we could add a link here?

Reply:
Yes, that's a good additional source of information for future contributors